docs/user-guide/migrating-to-5.0.0.md

```diff
@@ -194,6 +196,22 @@ When parsing JSX code like `<a>foo</a>`, the default parser will now give the `f
 
 **To address:** If you have written a custom rule that relies on text nodes in JSX elements having the `Literal` type, you should update it to also work with nodes that have the `JSXText` type.
 
+## <a name="spread-operators"></a> When using the default parser, spread operators now have type `SpreadElement`
+
+Previously, when parsing JS code like `const foo = {...data}` with the `experimentalObjectRestSpread` option enabled the default parser would set the `...data` AST node the `ExperimentalSpreadProperty` type.
```

Comment:
Could you please add a comma after "with the `experimentalObjectRestSpread` option enabled" (after the word "enabled")?

And I think the second half of the sentence might read more naturally like this:

<pre>the default parser would generate an `ExperimentalSpreadProperty` node type for the `...data` spread element.</pre>

(But I may be wrong here.)